Cannout find module rimraf

It looks like `rimraf` is listed as a dev dependency, but should be listed as a regular dependency. When running `ember exam:iterate`, it ends with 

```text
Cannot find module 'rimraf'
Error: Cannot find module 'rimraf'
    at Function.Module._resolveFilename (module.js:325:15)
    at Function.Module._load (module.js:276:25)
    at Module.require (module.js:353:17)
    at require (internal/module.js:12:17)
    at CoreObject.module.exports._cleanupBuild (/Users/xanderdumaine/Code/web-directory/node_modules/ember-exam/lib/commands/exam/iterate.js:84:18)
```

--- a/package.json
+++ b/package.json
@@ -47,7 +47,6 @@
     "istanbul": "^0.4.3",
     "loader.js": "^4.0.1",
     "mocha": "^2.3.4",
-    "rimraf": "^2.5.4",
     "rsvp": "^3.2.1",
     "sinon": "^1.17.5"
   },
@@ -59,7 +58,8 @@
     "cli-table2": "^0.2.0",
     "debug": "^2.2.0",
     "ember-cli-babel": "^5.1.6",
-    "fs-extra": "^0.30.0"
+    "fs-extra": "^0.30.0",
+    "rimraf": "^2.5.4"
   },
   "ember-addon": {
     "configPath": "tests/dummy/config"

